core/src/main/scala/org/bitcoins/core/number/NumberCache.scala

```diff
@@ -46,6 +46,11 @@ trait NumberCacheBigInt[T] extends NumberCache[T] {
   /** The max number cached (inclusive) */
   def maxCachedBigInt: BigInt = BigInt(maxCached)
 
+  /** [[org.bitcoins.core.protocol.CompactSizeUInt]] uses a UInt64
```

Comment:
Truthfully i'm still guessing with this number. It seems like most things with scriptSigs should be less than this size?